OutOfSchool/OutOfSchool.WebApi/Util/MappingProfile.cs

```diff
@@ -571,7 +571,9 @@ public MappingProfile()
         CreateMap<Rating, RatingDto>()
             .ForMember(dest => dest.FirstName, opt => opt.Ignore())
             .ForMember(dest => dest.LastName, opt => opt.Ignore());
+
         CreateMap<RatingDto, Rating>()
+            .ForMember(dest => dest.IsDeleted, opt => opt.Ignore())
```

Comment:
(DRY - again, I'm not insisting on changes) 
Please consider whether such helper method could ease repetitions
```
    public IMappingExpression<TSource, TDestination> CreateSoftDeletedMap<TSource, TDestination>()
        where TSource : class
        where TDestination : class, ISoftDeleted
        => CreateMap<TSource, TDestination>()
            .ForMember(dest => dest.IsDeleted, opt => opt.Ignore());
```
usage:
```
    CreateSoftDeletedMap<RatingDto, Rating>()
        .ForMember(dest => dest.Parent, opt => opt.Ignore());
```

Reply:
Done